### ISSUE TITLE
flutter 2.2.3 version ,ios14.5  error

### ISSUE THREAD

**Issue by gxzlolo** [creativecreatorormaybenot/wakelock#127]
Launching lib/main.dart on iPhone 8 in debug mode...
Xcode build done.                                           11.9s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    /Users/charles/.pub-cache/hosted/pub.flutter-io.cn/wakelock-0.5.2/ios/Classes/messages.m:4:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    In file included from /Users/charles/.pub-cache/hosted/pub.flutter-io.cn/wakelock-0.5.2/ios/Classes/WakelockPlugin.m:1:
    /Users/charles/.pub-cache/hosted/pub.flutter-io.cn/wakelock-0.5.2/ios/Classes/WakelockPlugin.h:1:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    1 error generated.
    In file included from /Users/charles/.pub-cache/hosted/pub.flutter-io.cn/wakelock-0.5.2/ios/Classes/WakelockPlugin.m:1:
    /Users/charles/.pub-cache/hosted/pub.flutter-io.cn/wakelock-0.5.2/ios/Classes/WakelockPlugin.h:1:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete
Could not build the application for the simulator.
Error launching application on iPhone 8.
Exited (sigterm)

**Comment by gxzlolo** [comment]
0.5.2 iOS

**Comment by creativecreatorormaybenot** [comment]
Duplicate of https://github.com/creativecreatorormaybenot/wakelock/issues/95.
